charts/redis-ha/templates/redis-ha-configmap.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-configmap
+  labels:
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app: {{ template "redis-ha.fullname" . }}
+data:
+  redis.conf: |
+    dir /data
+    maxmemory {{ .Values.redis.maxMemory }}
+    maxmemory-policy {{ .Values.redis.maxMemoryPolicy }}
+    min-slaves-max-lag {{ .Values.redis.minSlavesMaxLag }}
+    min-slaves-to-write {{ .Values.redis.minSlavesToWrite }}
+    repl-diskless-sync {{ .Values.redis.disklessReplication }}
+    rdbcompression yes
+    rdbchecksum yes
+    {{ .Values.redis.rdbBackups }}
+    {{ .Values.redis.addionalConfig }}
+
+  sentinel.conf: |
+    dir /data
+    sentinel monitor {{ .Values.redis.masterGroupName }} {{ template "redis-ha.fullname" . }}-0.{{ template "redis-ha.fullname" . }} 6379 {{ .Values.sentinel.quorum }}
```

Comment:
I can't tell if we are getting the bootstraplist here or just the first host.

Reply:
I copied Federation which initializes the conf with the first host as the master and then configures the rest as slaved by default.

Reply:
Yeah actually that's true. We are good then.